.github/workflows/main.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       # TODO: Split the linting into a separate job.
       - name: Run lint task
         # Only run this step for the default Node version.
-        if: matrix.node == env.DEFAULT_NODE_VERSION
+        if: matrix.node-version == env.DEFAULT_NODE_VERSION
```

Comment:
You don't need here `${{ matrix.node-version }} == ....` ?

Reply:
No, everything inside of `if` is considered an expression so there is no need to open an expression here manually.

Reply:
Ok, thanks for the explanation. I was just curious about it.